lessons/ro/chapter_8.yaml

```
@@ -0,0 +1,357 @@
+    
+    Singurul său scop este să reprezinte conceptual locul în care se află un 
+    
+    anumit tip de dată. Ceea ce deosebește o referință de un simplu număr este
+    
+    faptul că limbajul Rust se asigură că durata de viață a unei referințe nu este
+    
+    mai mare decât cea a valorii referențiate (altfel ar apărea erori când o folosim).
+- title: Pointeri simpli
+  code: >-
+    https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&code=fn%20main()%20%7B%0A%20%20%20%20let%20a%20%3D%2042%3B%0A%20%20%20%20let%20memory_location%20%3D%20%26a%20as%20*const%20i32%20as%20usize%3B%0A%20%20%20%20println!(%22Data%20is%20here%20%7B%7D%22%2C%20memory_location)%3B%0A%7D%0A
+  content_markdown: >
+    Referințele pot fi transformate într-un tip de date mai simplu, 
+    
+    numit *pointer simplu*. La fel ca un număr, acesta poate fi copiat si mutat
+    
+    fără prea multe restricții. Limbajul Rust nu ne asigură cu nimic de 
+    
+    valabilitatea zonei de memorie la care pointează.
+
+
+    Există două feluri de pointeri simpli:
+
+
+    * `*const T` - Un pointer simplu către un tip de date T care nu va fi modificat.
+
+    * `*mut T` - Un pointer simplu către un tip de date T ce poate fi modificat.
+
+
+    Pointerii simpli pot fi convertiți în și din numere (ex `usize`).
+
+
+    Pointerii simpli pot accesa date din cod *nesigur* (mai multe despre acest subiect mai târziu).
+
+
+    Detalii despre memorie:
+
+    * O referință în Rust este foarte asemănătoare cu un pointer din C în ceea
+    ce privește utilizarea, dar cu mult mai multe restricții privind cum poate 
+      fi stocat și folosit de alte funcții.
+    * Un pointer simplu în Rust este asemănător unui pointer din C ce reprezintă un număr care poate fi
+      copiat și pasat ca argument, ba chiar poate fi transformat într-o valoare numerică și modificat
+      pentru a face aritmetică pe pointeri.
+- title: Dereferențierea
+  content_markdown: |
+    Procesul prin care se accesează/utilizează datele care sunt referențiate 
+    de către o *referință* (adică `&i32`) se numește *dereferențiere*. 
+
+    Referințele sunt folosite pentru a accesa/utiliza datele în doua feluri:
+    * Accesează datele referențiate în timpul declarării variabilei.
+    * Accesează câmpurile sau metodele datelor referențiate.
+
+    Limbajul Rust are câțiva operatori utili care ne permit să facem asta.
+- title: Operatorul `*`
+  code: >-
+    https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&code=fn%20main()%20%7B%0A%20%20%20%20let%20a%3A%20i32%20%3D%2042%3B%0A%20%20%20%20let%20ref_ref_ref_a%3A%20%26%26%26i32%20%3D%20%26%26%26a%3B%0A%20%20%20%20let%20ref_a%3A%20%26i32%20%3D%20**ref_ref_ref_a%3B%0A%20%20%20%20let%20b%3A%20i32%20%3D%20*ref_a%3B%0A%20%20%20%20println!(%22%7B%7D%22%2C%20b)%0A%7D
+  content_markdown: >
+    Operatorul `*` reprezintă o metodă explicită pentru a dereferenția o referință. 
```

"Metodă" may be interpreted as a method instead of a way to dereference.
```suggestion
    Operatorul `*` reprezintă o modalitate explicită pentru a dereferenția o referință. 
```